projects/canopy/src/lib/details/details.component.html

```diff
@@ -4,6 +4,7 @@
   [id]="panelId"
   [class.lg-details__panel--active]="isActive"
   [attr.aria-labelledby]="toggleId"
+  [attr.role]="'navigation'"
```

Comment:
This doesn't seem to be the correct `role` to use:
```
The navigation role is used to identify major groups of links used for navigating through a website or page content.
```
I think it needs a bit more investigation.

Reply:
Nice one, thanks Elena, I'll put that one in 😉

Reply:
Updated the role.